package.json

```diff
@@ -37,10 +37,8 @@
     "eslint": "7.31.0",
     "eslint-config-next": "11.0.1",
     "eslint-plugin-jest": "^24.4.0",
-    "identity-obj-proxy": "^3.0.0",
     "jest": "^27.0.6",
-    "react-addons-test-utils": "^15.6.2",
-    "react-test-renderer": "^17.0.2",
+    "jest-css-modules": "^2.1.0",
```

Comment:
![image](https://user-images.githubusercontent.com/26348192/128978578-b936352a-df27-4961-9183-419620100079.png)
Is this package really required or could you implement similar functionality without using this package?

If it's actually required, then just comment below :)